.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -105,6 +105,9 @@ stages:
             echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
             echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
             echo '##vso[task.setvariable variable=OS_VERSION]16.04'
+            echo '##vso[task.setvariable variable=IMG_PUBLISHER]Canonical'
+            echo '##vso[task.setvariable variable=IMG_OFFER]UbuntuServer'
+            echo '##vso[task.setvariable variable=IMG_SKU]16.04-LTS'
```

Comment:
Just curious, why is 16.04 being added here, even though we don't support it now?

Reply:
We can revert that - I added it so that I could test more easily on a 1604 VHD, since even though it's not supported, we do still have clusters out there using it. It turns out the image build doesn't work for other reasons anyway, and I was able to test using the last built image.